psi4/driver/procrouting/proc.py

```diff
@@ -1220,15 +1220,9 @@ def scf_helper(name, post_scf=True, **kwargs):
         p4util.banner('  Computing high-spin triplet guess  ')
         core.print_out('\n')
 
-    # If GUESS is auto guess what it should be
+    # If GUESS is auto, it should probably be SAD
     if core.get_option('SCF', 'GUESS') == "AUTO":
-        if (core.get_option('SCF', 'REFERENCE') in ['RHF', 'RKS']) and \
```

Comment:
Can we keep this auto guess for now and switch one of the passing ROHF test cases to using SAD so that we can separate out the code integrity and the guess issue?

Reply:
Sure

Reply:
I've added a new scf-guess test for smoke tests of all guess types.